Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCallingSuccess.scala

```diff
@@ -1,7 +1,43 @@
 package nl.lumc.sasc.biopet.test.shiva.svcalling
 
-import nl.lumc.sasc.biopet.test.PipelineSuccess
+import java.io.File
 
-trait ShivaSvCallingSuccess extends ShivaSvCalling with PipelineSuccess { //TODO: extend SummaryPipeline
+import nl.lumc.sasc.biopet.test.{ SummaryGroup, SummaryPipeline }
+import nl.lumc.sasc.biopet.test.shiva.svcallers.{ Breakdancer, Clever, Delly }
+import org.testng.annotations.Test
 
+trait ShivaSvCallingSuccess extends ShivaSvCalling with SummaryPipeline {
+  @Test
+  def cleverDir: Unit = {
+    val dir = new File(outputDir, "clever")
+    if (svCallers.exists(_.isInstanceOf[Clever])) dir should exist
+    else dir shouldNot exist
+  }
+
+  @Test
+  def BreakdancerDir: Unit = {
+    val dir = new File(outputDir, "breakdancer")
+    if (svCallers.exists(_.isInstanceOf[Breakdancer])) dir should exist
+    else dir shouldNot exist
+  }
+
+  @Test
+  def dellyDir: Unit = {
+    val dir = new File(outputDir, "delly")
+    if (svCallers.exists(_.isInstanceOf[Delly])) dir should exist
+    else dir shouldNot exist
+  }
+
```

Comment:
I'd prefer 1 method instead of 3
`@Test(dataProvider = "callers")`
`def assertOutputExists(SvCaller svCaller): Unit = {`
`    val dir = new File(outputDir, svCaller.svCallerName)`
`    dir should exist`
`}`